test.js

```diff
@@ -1,13 +1,24 @@
 import test from 'ava';
 import fn from './';
 
-test(async t => {
+test('should scrape ~sindresorhus', async t => {
 	const user = await fn('sindresorhus');
 
-	t.is(user.name, 'Sindre Sorhus');
+	t.is(user.name, null);
```

Comment:
You can't just set this to `null`. If it returns `null` it's clearly something wrong going on in the existing code.

Reply:
Your name really is null.
- [yours](https://www.npmjs.com/~sindresorhus)
- [mine (with name)](https://www.npmjs.com/~charleskenney)

Reply:
Wut. That's super weird. Seems like npm had a bug which nulled all my info... I remember having all that set up at least half a year ago.

Reply:
I've added back the info now on npm, so you can revert these changes.